<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OddChangeDlg</class>
 <widget class="QDialog" name="OddChangeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OddChangeDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2 {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>157</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="id_lab_received">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>待收金额：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="receivedValueLable">
         <property name="geometry">
          <rect>
           <x>186</x>
           <y>14</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font:bold;
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>￥37.02</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="id_lab_fact">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>46</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>实收金额：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="factValueLabel">
         <property name="geometry">
          <rect>
           <x>186</x>
           <y>44</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font:bold;
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>￥10.00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="id_lab_odd_change">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>109</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>找零金额：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="oddChangeValueLabel">
         <property name="geometry">
          <rect>
           <x>186</x>
           <y>109</y>
           <width>130</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font:bold;
font-family:&quot;微软雅黑&quot;;
color: rgb(234, 136, 82);
font-size:22px;</string>
         </property>
         <property name="text">
          <string>￥27.02</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>3470</width>
           <height>2</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
